README.md

```diff
@@ -2,54 +2,120 @@
+Many users will likely prefer alternative ways to use the platform rather than working directly with the API. Therefore, to make Globalping accessible to all kinds of users, we're constantly expanding and improving our tools, all of which leverage the full potential of the Globalping API:
 
-Learn more about Globalping on our website [www.jsdelivr.com/globalping](https://www.jsdelivr.com/globalping)
+- [CLI tool](#globalping-cli) for people who feel at home in their terminal
+- [Web tools](#web-based-tools---globalping-website) for on-the-go testing and visual presentations
+- [Slack app](#slack-app) for bringing network testing capabilities into your Slack workspace
+- [GitHub bot](#github-bot) for adding network test results to public GitHub issues
 
-## Major sponsors
+Discover more about Globalping on our website [www.jsdelivr.com/globalping](https://www.jsdelivr.com/globalping)
 
-We thank our sponsors who contribute to the development of Globalping and help us expand the network of probes.
+## Our major donors
+We thank our donors who contribute to the development of Globalping and help us expand our probe network!
 
 | <img src="https://gcore.com/favicon.ico" width="15" height="15"> [Gcore](https://gcore.com) | <img src="https://xtom.com/favicon.ico" width="15" height="15"> [xTom](https://xtom.com) | <img src="https://www.edisglobal.com/favicon.png" width="15" height="15"> [Edis Global](https://www.edisglobal.com) |
 |---|---|---|
 
-Consider [running our probe software](https://github.com/jsdelivr/globalping-probe) on your network of servers or [sponsor a batch of hardware probes](https://docs.google.com/document/d/1xIe-BaZ-6mmkjN1yMH5Kauw3FTXADrB79w4pnJ4SLa4/edit#heading=h.tmcgof2zm3yh) with your own stickers and swag included to ship to your users or distribute at conferences.
-Stand out among the sea of water bottles and pens!
+### Support the Globalping community and platform 
+We welcome any individual or company interested in supporting Globalping's growth and our mission to make the internet a faster place for everyone. 
+Here are some ways you can help:
+
+- [Run multiple probes](https://github.com/jsdelivr/globalping-probe): Globalping relies on a globally distributed network of probes for running network measurement requests. We're happy to list anyone who can host at least six probes as a donor on GitHub and our website.
+- [Become a GitHub sponsor](https://www.jsdelivr.com/sponsors): By becoming a GitHub sponsor of [jsDelivr](https://github.com/jsdelivr), you are supporting both the jsDelivr CDN and the Globalping platform. Sponsors contributing $10 or more per month can request a hardware probe to install in their home or office.
```

Comment:
jsDelivr sponsors are unrelated to GP sponsors, lets link here instead to avoid confusion https://github.com/sponsors/jsdelivr

Reply:
I replaced the first link -- hope that's what you meant

Reply:
I dont think you did, I dont see any changes.

Reply:
Thank you -- now it's updated!